dashboard/app/helpers/users_helper.rb

```diff
@@ -116,17 +116,25 @@ def log_self_takeover_investigation_to_firehose(source_user:, destination_user:,
   #     "135": {"status": "perfect", "result": 100}
   #   }
   # }
-  def summarize_user_progress(script, user = current_user, exclude_level_progress = false)
-    user_data = user_summary(user)
-    merge_script_progress(user_data, user, script, exclude_level_progress)
```

Comment:
style point (for next time): In general, extracting (or in this case not inlining) methods improves readability since it makes the parent method shorter and more self-documenting. I'm open to the argument that inlining this particular method is more readable in this case, but what I would NOT recommend is automatically inlining any method which only appears in one place.